gwcs/models.py

```diff
@@ -0,0 +1,127 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, unicode_literals, print_function
+
+import numpy as np
+
+from astropy.modeling.core import Model
+from astropy.modeling.parameters import Parameter
+
+try:
+    from scipy.interpolate import interpn
```

Comment:
Good point. I'll add a version check. The latest is scipy v 0.16.1. `interp1d` does not support extrapolation so I'd prefer not to fall back on it unless upgrading scipy becomes a serious issue .